docs/execution-providers/Cloud-ExecutionProvider.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Cloud
+description: Instructions to infer an ONNX model remotely with a cloud endpoint
+parent: Execution Providers
+nav_order: 12
+redirect_from: /docs/reference/execution-providers/Cloud-ExecutionProvider
+---
+
+# Cloud Execution Provider
```

Comment:
Better to mention Preview in parentheses here.

Reply:
Done.